### DISCUSSION TITLE
Chromium Update to v117.0.5938.89 causing error code 419

### DISCUSSION THREAD

**Discussion by JF-626164** [laravel/framework#48442] [Q&A]
Hello all! I need some help.

On Friday, 9/15/23, Edge & Chrome updated to version 117.0.5938.89. Since that update my users cannot log into the site as it is throwing error code 419 on page load and constantly pops up an alert message saying quote: "This page has expired. Would you like to refresh the page?"

Refreshing the page does not fix this issue.

There was no code change in my project between before the v117 update and today that could've caused this issue. To that end, Firefox works just fine. 

I've dug into the code and it's a csrf_token mismatch issue.

On the login blade, I'm using a form post to send the users login details to the controller. In it is "@csrf" as mentioned in the laravel documentation. So it's getting and using the correct token. But when the login button is pressed, all the way in the vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken, in the tokensMatch function: The $token variable and the $request->session()->token() are not the same.

And I don't know why, or how to fix.

Any thoughts?

**Comment by crynobone** [comment]
Duplicate of https://github.com/laravel/framework/issues/48445
